assets/src/js/_acf.js

```diff
@@ -673,9 +673,8 @@
 	 *  @return	string Translated text.
 	 */
 
-	if ( window.acfL10n == undefined ) {
-		acfL10n = {};
-	}
+	// Make sure a global acfL10n object exists to prevent errors in other scopes
```

Comment:
Outside the scope of this PR, but we should probably add some checks on coding style outside PHP files, e.g. to create a consistent way for commenting etc (https://github.com/WordPress/secure-custom-fields/pull/118/files#diff-2c39fc5eb9675f3ca4efecbc4a219c3c560c52ad26d42d45217ad878755a59dfL852 - note the ending stop symbol)

Reply:
That's a good catch, thanks for noticing 💯

Reply:
It's not the only inconsistency, don't worry!